src/topp/OpenSwathAssayGenerator.cpp

```diff
@@ -236,7 +237,7 @@ class TOPPOpenSwathAssayGenerator :
 
       if (enable_ms1_uis_scoring)
       {
-        for (int i = 0; i < round((precursor_upper_mz_limit - precursor_lower_mz_limit) / precursor_mz_threshold); i++)
+        for (int i = 0; i < Math::round((precursor_upper_mz_limit - precursor_lower_mz_limit) / precursor_mz_threshold); i++)
```

Comment:
minor comment:
this will give a comparison warning (int vs double) and since the condition is eval'ed every iteration,
You could
```
int thresh = static_cast<int>(Math::round((precursor_upper_mz_limit - precursor_lower_mz_limit) / precursor_mz_threshold));
for ....
```